webApp/public/js/script.js

```diff
@@ -1141,19 +1141,6 @@
 			ParallaxScroll.init();
 		}
 
-		// Select 2
-		if ( plugins.selectFilter.length ) {
-			for ( var i = 0; i < plugins.selectFilter.length; i++ ) {
-				var select = $( plugins.selectFilter[ i ] );
-
-				select.select2( {
-					placeholder:             select.attr( 'data-placeholder' ) || null,
-					minimumResultsForSearch: select.attr( 'data-minimum-results-search' ) || Infinity,
-					containerCssClass:       select.attr( 'data-container-class' ) || null,
-					dropdownCssClass:        select.attr( 'data-dropdown-class' ) || null
-				} );
-			}
-		}
 
```

Comment:
Keep for now (other fix in different branch)
```suggestion
		// Select 2	
		if ( plugins.selectFilter.length ) {	
			for ( var i = 0; i < plugins.selectFilter.length; i++ ) {	
				var select = $( plugins.selectFilter[ i ] );	

				select.select2( {	
					placeholder:             select.attr( 'data-placeholder' ) || null,	
					minimumResultsForSearch: select.attr( 'data-minimum-results-search' ) || Infinity,	
					containerCssClass:       select.attr( 'data-container-class' ) || null,	
					dropdownCssClass:        select.attr( 'data-dropdown-class' ) || null	
				} );	
			}	
		}
		```